rfc006-release-planning-mgt.md

```diff
@@ -0,0 +1,143 @@
+
+- The Templates collection
+- The Doc Tools initiatives
+- The Chronologue project, by virtue of its inclusion as examples to both the above
+
+Note however that while a given Release from The Good Docs Project may not include any *updated* material as part of a Product Output, its current version will still be included as part of any release packages that are creation. For example, the Doc Tools Working Group may release a new version of that software as part of a Release implementing a new feature. But the Chronologue WG's content included with that software need not be updated at all, unless the new feature specifically requires it (e.g. it makes use of a new metadata field).
+
+
+## Roles and Responsibilities
+
+The main participants in the Release Planning Process are as follows:
+
+- Project Steering Committee: The PSC will collectively create goals for releases, and 
+- Release Manager: This role will be responsible for driving the planning process for future Releases, and for maintaining an easy-to-understand status on any in-process Release(s). This individual will interact regularly with Working Group leaders to understand current progress, and update the PSC if it's trending away from targets.
+- Working Group Coordinator: Each Working Group should designate a coordinator, who will work with the Release Manager in reviewing the goals of the Release and determining if they are feasible given available time and volunteer bandwidth. The will also provide updates on the WG's progress during the course of a Release. While Coordinators will be responsible for gathering the progress of their WG's members, the Release Manager will establish communication channels that will make this efficient.
+
+
+## Proposal
+
+The release planning process will be a collaborative process between members, their Working Groups, and the Project Steering Committee, and the Release Manager as follows:
+
+1. The PSC will decide on a target date for an upcoming release. This can be driven by the desire to release together with a particular event (e.g. at a Write The Docs conference), or based on a regular schedule (such as quarterly). The overall goals of the project release will be tracked in a TBD location: ZenHub if it supports recursive Epics, or e.g. a wiki if not.
+1. Based on a target release date, Working Groups will convene and decide (based on capacity) the items to which they can commit. They will be able to plan the precise issues they will complete as part of a release using their workspace in the ZenHub application.
+1. The PSC will evaluate each Working Group's release plan and provide feedback. As the plan will be based on a realistic bandwidth estimate, the PSC should focus on identifying items to be added/removed based on strategic priorities. The PSC and Working Groups will negotiate and come to a consensus on what from each sub-project will be included in the release.

```

Comment:
As above, I feel the PSC's role should only be oversight, only needing to review, comment and then approve proposals put to them.

Day to day work, such as reviewing a working group's proposal should be done by the release manager (who may defer to the PSC for a contentious decision if needed).

If the release manager job becomes too big for one person, we could spawn a Release Management Committee in order to spread the load and decision making.

(This comment also applies to other lines in this Proposal section.)